### PR TITLE
Css box model - Sina Daryaei

### PR DESCRIPTION
**Pull request by sinadaryaei**
image

### CODE REVIEW

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change requested here Sina!

File: css-box-model/styles.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+  /* your code here 👇 */
+}
+
+.language {
+  margin-left: 12px;
+  margin-right: 16px;
+  padding-top: 6px;
+  padding-bottom: 6px;
+  border-radius: 6px;
```
[the remark this answers is not in the document]
Comment (sinadaryaei) [reply css-box-model/styles.css:13]: Can I use padding as shorthand?

File: css-box-model/styles.css [hunk 2]
```diff
@@ -8,7 +8,6 @@ button.language {
 .language {
   margin-left: 12px;
   margin-right: 16px;
-  padding-top: 6px;
-  padding-bottom: 6px;
+  padding: 10px 15px 20px 25px;
```
Comment (shawnkost) [inline css-box-model/styles.css:11]: Your padding values are incorrect. Look back at the message I sent you last night

**Review by shawnkost — CHANGES_REQUESTED** [review]
The padding values are incorrect here! Top / bottom should have the same value, same with left / right

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**